src/hatti/ona/post_process.cljs

```diff
@@ -98,6 +99,21 @@
      :download_url file-url
      :small_download_url (str file-url "&suffix=small")}))
 
+(defn get-matching-name
+  "Gets and returns matching name for an attachment that was renamed by
+  appending a hash on the S3 server to avoid duplicating names.
+  e.g. if 1478203839187.jpg was renamed to 1478203839187_wijUzUf.jpg,
+  it returns 1478203839187_wijUzUf.jpg."
+  [fname fnames]
+  (if (string? fname)
```

Comment:
hmm, tricky, something feels overly complicated here. so flagging but I can't think of how to simplify it right now